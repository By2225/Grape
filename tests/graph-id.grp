fun Int main() {
        Graph<Int,Int> a;
        Node<Int> b;

        b = '3';

        a = << b -4- '5', b -10- '6' >>;

        return 0;
}
