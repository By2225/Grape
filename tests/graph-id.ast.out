
fun Int main() {
Graph<Node<Int>, Edge<Int>> a;
Node<Int> b;
b = '3';
a = <<b -4- '5' , b -10- '6' >>;
return 0;
}
